src/Summoner/Template.hs

```diff
@@ -536,15 +536,15 @@ createProjectTemplate ProjectData{..} = Dir (toString repo) $
               export PATH="/opt/ghc/$$GHCVER/bin:/opt/cabal/$$CABALVER/bin:$$PATH"
               echo $$PATH
               cabal new-update
-              cabal new-build --enable-tests --enable-benchmarks
+              cabal new-build --enable-tests --enable-benchmarks --ghc-options=-Werror
```

Comment:
Unfortunately, this doesn't work for `cabal` 😞  See relevant issue: https://github.com/haskell-CI/haskell-ci/issues/95

You can either use the trick mentioned in the issue or just leave `-Werror` only for `stack` and create another issue in `summoner`.

Reply:
And here is relevant `cabal` issue:

* https://github.com/haskell/cabal/issues/3883